.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -23,4 +23,15 @@ test/README.md and add these additional items:
 - [ ] QE has approved reproducer/new tests and
 - [ ] Subsequent commits provide bug fixes without modifying the reproducer; CI reports PASS and
 - [ ] QE approves this PR; RHBZ status is set to `MODIFIED + Verified=Tested`
+
+Information regarding our CI:
+
+CI will not be ran automatically if WIP label is applied or the PR is in DRAFT state, instead only
+a link will be provided to the pipeline which can then be triggered manually if desired. To run the
+CI automatically either switch the PR to ready or apply WIP+test label.
+
+Outside contributors need manual approval from one of the osbuild team members or request to
+be added to "friends of osbuilders" team for the CI to trigger without approval.
+
+Also CI is triggered after Tests actions are done for the PR which include Lint, Unit tests and ShellCheck.
```

Comment:
I suggest not enumerating all tests. It can get outdated very quickly (it actually already is, we have 5 checks now). Mentioning that all Tests jobs in the github workflow must succeed in order to the gitlab pipeline to get triggered is enough.